tools/tools.go

```diff
@@ -36,6 +36,7 @@ import (
 
 //go:generate go run check.go
 
+//go:generate go build -v -o ../bin/ check-switch/check-switch.go
```

Comment:
```suggestion
//go:generate go build -v -o ../bin/ ./check-switch
```

To use package path as in other lines